sympy/calculus/util.py

```diff
@@ -373,6 +375,16 @@ def periodicity(f, symbol, check=False):
 
         period = _periodicity(g.args, symbol)
 
+    elif isinstance(f, Mod):
+        a,n=f.args
```

Comment:
space after comma
space around `==`
space around  `=` except in arg list
(see [pep8](https://www.python.org/dev/peps/pep-0008/))